index.html

```diff
@@ -31,6 +31,192 @@ <h3>I am a Software Developer</h3>
+		<div class="data">
+			<h2 class="datatext">Profesional Art Printing Data</h2>
+			<p>A daily selection of privately personalized reads; 
+				no accounts or sign-ups required. has been 
+				the industry's standard</p>
+
+				<ul class="btn">
+					<li class="btn-skills">html</li>
+					<li class="btn-skills">bootstrap</li>
+					<li class="btn-skills">Ruby</li>
+				</ul>
+	
+			<ul class="btn-mid">
+				<li class="btn-see-project"><a href="#">See Project</a></li>
+			</ul>
+		</div>
+	</section>
+  
+  <section>  
+  <div class="data">
+		<h2 class="datatext">Profesional Art Printing Data</h2>
+		<p>A daily selection of privately personalized reads; 
+			no accounts or sign-ups required. has been 
+			the industry's standard</p>
+
+			<ul class="btn">
+				<li class="btn-skills">html</li>
+				<li class="btn-skills">bootstrap</li>
+				<li class="btn-skills">Ruby</li>
+			</ul>
+
+		<ul class="btn-mid">
+			<li class="btn-see-project"><a href="#">See Project</a></li>
+		</ul>
+	</div>
+	</section>
+
+	<section>
+		<div class="data">
+			<h2 class="datatext">Profesional Art Printing Data</h2>
+			<p>A daily selection of privately personalized reads; 
+				no accounts or sign-ups required. has been 
+				the industry's standard</p>
+
+				<ul class="btn">
+					<li class="btn-skills">html</li>
+					<li class="btn-skills">bootstrap</li>
+					<li class="btn-skills">Ruby</li>
+				</ul>
+	
+			<ul class="btn-mid">
+				<li class="btn-see-project"><a href="#">See Project</a></li>
+			</ul>
+		</div>
+	</section>
+
+	<section>
+		<div class="data">
+			<h2 class="datatext">Profesional Art Printing Data</h2>
+			<p>A daily selection of privately personalized reads; 
+				no accounts or sign-ups required. has been 
+				the industry's standard</p>
+
+				<ul class="btn">
+					<li class="btn-skills">html</li>
+					<li class="btn-skills">bootstrap</li>
+					<li class="btn-skills">Ruby</li>
+				</ul>
+	
+			<ul class="btn-mid">
+				<li class="btn-see-project"><a href="#">See Project</a></li>
+			</ul>
+		</div>
+	</section>
+	<section>
+		<div class="data">
+			<h2 class="datatext">Profesional Art Printing Data</h2>
+			<p>A daily selection of privately personalized reads; 
+				no accounts or sign-ups required. has been 
+				the industry's standard</p>
+
+				<ul class="btn">
+					<li class="btn-skills">html</li>
+					<li class="btn-skills">bootstrap</li>
+					<li class="btn-skills">Ruby</li>
+				</ul>
+	
+			<ul class="btn-mid">
+				<li class="btn-see-project"><a href="#">See Project</a></li>
+			</ul>
+		</div>
+	</section>
+	<section>
+		<div class="data">
+			<h2 class="datatext">Profesional Art Printing Data</h2>
+			<p>A daily selection of privately personalized reads; 
+				no accounts or sign-ups required. has been 
+				the industry's standard</p>
+
+				<ul class="btn">
+					<li class="btn-skills">html</li>
+					<li class="btn-skills">bootstrap</li>
+					<li class="btn-skills">Ruby</li>
+				</ul>
+	
+			<ul class="btn-mid">

```

Comment:
- [ ]  Avoid this tag (ul) for a list that is not a list. You can tag this as a div.